templates/engage/kubernetes-event-us.md

```diff
@@ -0,0 +1,87 @@
+---
+wrapper_template: "engage/_base_engage_markdown.html"
+context:
+  title: "Kubernetes from Cloud to Edge: A Virtual Event"
+  meta_description: "From public cloud, to containers, and even lightweight devices, discover strategies for optimising your Kubernetes deployment and operations. Join us live on 31 July 2020."
+  meta_image: https://assets.ubuntu.com/v1/5098807a-85410502-1a8a1900-b55f-11ea-9e1a-e5edf86735e7.jpg
+  meta_copydoc: https://docs.google.com/document/d/1LClShZkNAZYCHhXah452SEFYdwetyPstodwRL0Y9ZV8/edit
+  header_title: "Kubernetes from Cloud to Edge"
+  header_subtitle: "On 31 July, learn how to streamline your Kubernetes deployment and operations from Canonical's experts."
+  header_image: "https://assets.ubuntu.com/v1/67b862a1-k8s+microk8s.svg"
+  header_image_width: "419"
+  header_image_height: "172"
+  header_url: "https://www.brighttalk.com/webcast/6793/421884?utm_source=Canonical&utm_medium=brighttalk&utm_campaign=421884"
+  header_cta: Register now
+  header_cta_class: p-button--positive
+  header_cta_all_sizes: true
+  header_class: p-engage-banner--dark
+  header_lang: en-us
+  webinar_code: '<div class="jsBrightTALKEmbedWrapper" style="width:100%; height:100%; position:relative;background: #ffffff;"><script class="jsBrightTALKEmbedConfig" type="application/json">{ "channelId" : 6793, "language": "en-US", "commId" : 421884, "displayMode" : "standalone", "height" : "auto" }</script><script src="https://www.brighttalk.com/clients/js/player-embed/player-embed.js" class="jsBrightTALKEmbed"></script></div>'
+---
+
+## About the event
+
+**Date and time**
+31 July, 2020
+[2:30-4:30pm CT]
+
+**What it is**: A live, interactive virtual workshop with Canonical’s Kubernetes experts. This will include a presentation from our speakers, panel discussion incorporating live comments and questions from the audience, and free Kubernetes resources and demos to help you achieve your technology goals.
+
+**Who should attend**: Developers and enterprise IT teams
+wanting to learn more about how to overcome common Kubernetes challenges in a variety of contexts, and the most efficient solutions for executing their Kubernetes strategy.
+
+**What you will learn**: Best tools and strategies for streamlining your Kubernetes deployment and operations from public cloud, to containers, and even lightweight devices. We will also discuss the Kubernetes portfolio of Canonical and Ubuntu, and how its main components - MicroK8s and Charmed Kubernetes - can contribute to the flexibility you need deployment at every stage of your container DevOps cycle.
+
```

Comment:
```suggestion
## Speakers:
```

Reply:
As it is you can easily infer these people are speakers, but thought this would make it explicit